src/forms/components/encounter/ohri-encounter-form.tsx

```diff
@@ -103,8 +110,11 @@ export const OHRIEncounterForm: React.FC<OHRIEncounterFormProps> = ({
     // set Formik initial values
     if (encounter) {
       allFormFields.forEach(field => {
-        const existingVal = getHandler(field.type)?.getInitialValue(encounter, field, allFormFields);
-
+        const handler = getHandler(field.type);
+        let existingVal = getHandler(field.type)?.getInitialValue(encounter, field, allFormFields);
```

Comment:
@samuelmale why do you need to call `getHandler(field.type)` if you already have the `handler` variable?

Reply:
Nice catch